src/Controller/UserController.php

```diff
@@ -107,6 +121,119 @@ public function getManageUsersAction()
+            $this->userEduArray = $users->getUserEducation();
+
+            $form = $this->createFormBuilder($user)
+                ->add('userFirstName', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $users->getUserFirstName()
+                ))
+                ->add('userLastName', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $users->getUserLastName()
+                ))
+                ->add('userEmail', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $userEmailString
+                ))
+                ->add('userMobileNumber', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $userMobileString
+                ))
+                ->add('userDateofBirth', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $users->getUserDateofBirth()
+                ))
+                ->add('userEducation',ChoiceType::class,array(
+                    'choices' => $educationArray,
+                    'choice_attr' => function($educationArray, $key, $value) {
+                        $selectVal = "";
+                        if(in_array($key, $this->userEduArray)){
+                            $selectVal = "selected";
+                        }
+                        return ['selected' => $selectVal];
+                    },
+                    'multiple' => true, 'attr' => array('class' => 'form-control'), 
+                ))
+                ->add('userBloodGroup', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $users->getUserBloodGroup()
+                ))
+                ->add('userGender', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $users->getUserGender()
+                ))
+                ->add('id', HiddenType::class, array(
+                    'data' => $id,
+                ))
+                ->getForm();
+
+            $form->handleRequest($request);
+            if ($form->isSubmitted() && $form->isValid()) {
+                $data = $form->getData();
+
+                $dm = $this->get('doctrine_mongodb')->getManager();
+                $user = $dm->getRepository(User::class)->find($data->getId());
+
+                $userEmailArray  = explode(",",$data->getUserEmail());
+                $userMobileArray = explode(",",$data->getUserMobileNumber());
+
+                $user->setUserFirstName($data->getUserFirstName());
+                $user->setUserLastName($data->getUserLastName());
```

Comment:
this is also not a recommended approach in symfony

Reply:
I have modified the symfony way approach.